### PR TITLE
CFE-3781: Deleted extra cruft from the masterfiles build

### PR DESCRIPTION
**Pull request by nickanderson**
Merge together:
https://github.com/cfengine/masterfiles/pull/2107

Before:
```
❯ ls -a out/masterfiles
.               templates            configure         MPF.md
..              tests                configure.ac      promises.cf
autom4te.cache  travis-scripts       CONTRIBUTING.md   promises.cf.in
cfe_internal    aclocal.m4           example_def.json  README.md
controls        .CFVERSION           .gitignore        standalone_self_upgrade.cf
.git            CFVERSION            install-sh        standalone_self_upgrade.cf.in
inventory       CHANGELOG.md         LICENSE           test-driver
lib             CODE_OF_CONDUCT.org  .mailmap          .travis.yml
m4              config.guess         Makefile          update.cf
misc            config.log           Makefile.am
modules         config.status        Makefile.in
services        config.sub           missing
```

After:
```
❯ ls -a out/masterfiles
.   cfe_internal  inventory  modules   templates    standalone_self_upgrade.cf
..  controls      lib        services  promises.cf  update.cf
```

### CODE REVIEW

**Comment by nickanderson** [comment]
@olehermanse what do you think, should we cull the other extra bits, Changelog, code of conduct, contributing, license, mpf.md, readme.md ?

**Comment by nickanderson** [comment]
@olehermanse  I went ahead and added another commit to cleanup the other bits, just lmk what you think we should keep from that or not.

File: index.json [hunk 1]
```diff
@@ -93,7 +93,38 @@
       "repo": "https://github.com/cfengine/masterfiles",
       "by": "https://github.com/cfengine",
       "commit": "c39b79c0e7a42522c69cff3d87a5bc5ac9471369",
-      "steps": ["run ./autogen.sh", "delete ./autogen.sh", "copy ./ ./"]
+      "steps": [
```
Comment (craigcomstock) [inline index.json:96]: do you also need to get rid of `.git` or any other hidden files/dirs at that top level? Like `.travis.yml`, `.CFVERSION`, `.gitignore`, `.mailmap` maybe?

File: index.json [hunk 2]
```diff
@@ -93,7 +93,44 @@
...
+          "delete ./CHANGELOG.md",
+          "delete ./CONTRIBUTING.md",
+          "delete ./LICENSE",
+          "delete ./MPF.md",
+          "delete ./.CFVERSION",
+          "delete ./.gitignore",
+          "delete ./.mailmap",
+          "delete ./.git",
+          "delete ./.travis.yml",
+          "delete ./controls/update_def.cf.in",
```
Comment (olehermanse) [inline index.json:131]: As indicated in slack, I think this list should be in the masterfiles repo, not in the index / cfbs json. We can put a `cleanup.sh` script in masterfiles:

```suggestion
          "run cleanup.sh",
          "delete cleanup.sh",
```

**Review by olehermanse — CHANGES_REQUESTED** [review]

**Comment by nickanderson** [comment]
@olehermanse as you wish.

**Pull request merged**